src/packages/README.md

```diff
@@ -26,33 +90,76 @@ the [CheriBSD ports issue tracker](https://github.com/CTSRD-CHERI/cheribsd-ports
 
 **CheriABI packages** are compiled using pure-capability CHERI C/C++, and
 employ fine-grained C/C++ memory protection.
-There are currently over 8,000 CheriABI packages available.
-These packages are considered *experimental*.
 
-These packages are installed in the standard `/usr/local` hierarchy.
+These packages are considered *appropriate for experimental use*.
+Their primary function is to provide necessary dependencies for efforts to port
+software to CheriABI and to support CHERI demonstrations.
+They are suitable for research and development of software to provide spatial
+and temporal memory safety as well as software compartmentalisation.
+They can be also used to investigate potential memory safety issues in
+third-party software that are easier to detect and debug using a CHERI-enabled
+hardware-software stack.
+
+There are currently over 8,000 CheriABI packages available, including:
+* Development utilities: `git`;
+* Editors: `nano`;
+* Libraries: `libnghttp2`, `libressl`, `libssh2`, `lzlib`, `wolfssl`;
+* Networking software: `rsync`;
+* Shells: `dash`.
+
+The packages are installed in the standard `/usr/local` hierarchy as they match
+the base system ABI.
 
 *Note:* These packages compile, but many have CHERI-related warnings that
-have not been audited and only a limited set have been tested.  Bugs
-related to CHERI support can be reported in the
-[CheriBSD ports issue tracker](https://github.com/CTSRD-CHERI/cheribsd-ports/issues).
+have not been audited and only a limited set have been tested. Bugs related to
+CHERI support can be reported in the [CheriBSD ports issue
+tracker](https://github.com/CTSRD-CHERI/cheribsd-ports/issues).
+While the CheriABI packages are more interesting as they use CHERI memory-safety
+features, you must remember that they might break in run-time and hence might
+not be suitable for critical operations.
+In such case, a corresponding hybrid ABI package might be considered instead of
+a CheriABI package.
 
-## Managing packages in CheriBSD
+## Duplicate packages
 
-*Note:* As of this writing we only provide packages for Morello systems.
-We aim to add RISC-V package sets in the near future.
+Hybrid ABI and CheriABI package repositories include many duplicate packages,
+e.g. `git`.
```

Comment:
I wonder if the word “duplicate” could be confusing -- they aren’t really duplicated, they are just the same software package compiled for both ABIs?